Formula/ucloud.rb

```diff
@@ -0,0 +1,21 @@
+class Ucloud < Formula
+  desc "The official tool to managment your ucloud services"
+  homepage "https://www.ucloud.cn"
+  url "https://ucloud-sdk.dl.ufileos.com/ucloud-cli-0.1.2.tar.gz"
+  sha256 "f41aaa6bf8063b3ccd0c5e41d3dc90f46f9fb45066513da8e306456f1f332071"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/ucloud/ucloud-cli"
+    dir.install buildpath.children
+    cd dir do
+      system "go", "build", "-o", bin/"ucloud"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Reply:
Thanks for your time!

Reply:
Thanks for the quick fix!